test/test_staff.py

```diff
@@ -61,29 +80,51 @@ def tearDown(self):
         test_models.DeleteAll(self.keys)    
 
     def _get(self, path):
-        return app.get(path, headers={'x-rooms-dev-signin-email': 'rebuildingtogether.staff@gmail.com'})
+        return APP.get(path, headers={'x-rooms-dev-signin-email': 'rebuildingtogether.staff@gmail.com'})
+
+    def _post(self, path):
+        return APP.post(path,
+                        headers={'x-rooms-dev-signin-email': 'rebuildingtogether.staff@gmail.com'},
+                        params={'submit': staff.EXPORT_CSV})
         
-    
+
 class StatefulTestStaffWithProgramAuto(StatefulTestStaffWithProgram):
     """Automatically test all routes to ensure they don't crash."""
     _multiprocess_can_split_ = True
 
-def _makerOfTestFunction(path):
```

Comment:
migrated this to a static method in the test class it's seeding, see get_test_function()